tests/configs/yaml/hbs/amqp/key.pem

```diff
@@ -0,0 +1,52 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
:exclamation:**Cycode**: Secret of type: 'Private Key' was found.

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_revoked** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_ignore_everywhere** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_ignore_here** | Applies to this request only |

 <!-- ab6083fd-3b33-4bd6-8a52-54ef2f7e90b6 -->

Reply:
#cycode_secret_ignore_here

Reply:
:+1: Cycode has finished processing your request. Please review your status checks.